src/js/components/RoutedAnchor/RoutedAnchor.js

```diff
@@ -14,6 +14,11 @@ class RoutedAnchor extends Component {
 
   render() {
     const { path, method, ...rest } = this.props;
+    if (process.env.NODE_ENV !== 'production') {
+      console.warn(
+        `This component will be deprecated in the 3.0 release. Please refer to https://github.com/grommet/grommet/issues/2855 for more information.`,
```

Comment:
I think we should avoid explicitly noting on which version the component will be deprecated.

Reply:
right makes sense. Should we mention next major release in this case? to note it will be a breaking change?

Reply:
let's say it will be deprecated on next upcoming releases?